package.json

```diff
@@ -1,8 +1,8 @@
 {
   "private": true,
-  "packageManager": "pnpm@8.15.3",
+  "packageManager": "pnpm@9.15.4",
```

Comment:
Bump required `pnpm` version to [`9.15.4`](https://github.com/pnpm/pnpm/releases/tag/v9.15.4)